Packs/CommonPlaybooks/ReleaseNotes/2_4_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Calculate Severity By Highest DBotScore
+
+Improved the performance of the playbook by ensuring that only unique indicators are retrieved from the cache.
```

Comment:
```suggestion
Improved the performance of the playbook by ensuring that unique indicators were retrieved from the context.
```

Reply:
I'll keep it as is because we're grabbing them from the cache here, not from context :)